features/support/hooks.rb

```diff
@@ -1,3 +1,28 @@
+Before('@ds_testing_table') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.